src/pages/Offer/sections/HedvigInfo.tsx

```diff
@@ -57,36 +73,33 @@ const Title = styled('h3')({
   margin: '0px',
   fontSize: '14px',
   color: colors.BLACK,
-  maxWidth: '240px',
 })
 
 const Paragraph = styled('p')({
   fontSize: '14px',
   color: colors.DARK_GRAY,
-  maxWidth: PARAGRAPHWIDTH,
 })
 
 const COLUMNS = [
   {
     key: 0,
-    title: 'Anmäl skador och få betalt på nolltid',
-    paragraph:
-      'Med vår smarta chattbot baserad på AI och maskininlärning kan vi snabbt göra bedömningar av skador och betala ut pengarna till dig utan krångliga formulär eller väntetider.',
-    image: '/assets/offering/placeholder.svg',
+    title: 'Enkel överblick',
```

Comment:
These texts are in GraphCMS and should be edited there (see #51 )

Reply:
Nice!